packages/website/docusaurus.config.js

```diff
@@ -57,7 +57,11 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/theme-common').UserThemeConfig} */
     ({
-      // sidebarCollapsible: false,
+      algolia: {
+        appId: 'BH4D9OD16A',
+        apiKey: '1ad6b47d4e742c4c0653877b5511c602',
+        indexName: 'typescript-eslint',
+      },
```

Comment:
@Josh-Cena is there something obviously wrong that I'm missing here...? No search is showing up 😞

Reply:
Well yeah, you didn't [add `theme-algolia` to the config](https://docusaurus.io/docs/next/using-themes#using-themes). If you are use `preset-classic` this should be done automatically, but I'm a bit surprised (also asked in #4157) that you didn't actually use the preset

Reply:
Aha thank you, now it all clicks! That fixed it.

Yeah I don't have a personal preference for or against preset-classic -- but it makes sense why we could use it to standardize+simplify...